sdk/digitaltwins/Azure.DigitalTwins.Core/src/Azure.DigitalTwins.Core.csproj

```diff
@@ -51,5 +51,11 @@
     </Compile>
   </ItemGroup>
 
-  <Import Project="$(RepoRoot)\sdk\core\Azure.Core\src\Azure.Core.props" />
+  <!--Uncomment the below to take reference from Azure.Core based on how Azure.Core.props defines how libraries should take reference-->
```

Comment:
Can you elaborate on the problem here?

Reply:
I can add more comments to the code itself, too, but the issue here is that the Azure.Core library hasn't had a release since they added the JsonPatchDocument class. So the only way we'll get access to it is to take a project reference instead.

This all becomes moot if we don't want the JsonPatchDocument in our API shape, though

Reply:
Gotcha. Might put it in those words as the existing comment didn't make sense to me.